resources/views/components/category-badge.blade.php

```diff
@@ -0,0 +1,7 @@
+@props(['categoryId'])
+
+<a class="text-sm border-2 border-red-500 cursor-pointer hover:bg-red-500 hover:text-white px-4 text-red-500"
+   href="/?category={{$categoryId}}"
```

Comment:
Using query params doesn't work. It's not implemented yet or it's a mistake made?

Reply:
I was going to implement query scopes for filtering, but decided not to.